waltz-common/src/main/java/com/khartec/waltz/common/IOUtilities.java

```diff
@@ -62,6 +62,15 @@ public static void copyStream(InputStream input, OutputStream output)
     }
 
 
+    /**
+     * Attempts to locate <code>fileName</code> via either:
+     * <ul>
+     *     <li>root of classpath</li>
+     *     <li>directory: <code>${user.home}/.waltz</code></li>
+     * </ul>
+     * @param fileName file (or path) to be located
+     * @return Resource representing the file or <code>null</code> if not found
```

Comment:
Minor comment: the return value is always non-null, `Resource.exists()` might be true or false

Reply:
Thanks. too early for coding!

Reply:
ha ha, true.. thanks for the PR!